modules/tinymce/src/core/test/ts/browser/init/InitEditorLanguageTest.ts

```diff
@@ -22,6 +22,6 @@ describe('browser.tinymce.core.init.InitEditorLanguageTest', () => {
   it('TBA: Should have been able to load a custom language pack', () => {
     UiFinder.notExists(SugarBody.body(), '.tox-notification');
     assert.equal(I18n.getCode(), 'custom', 'I18n language code should be the custom language code');
-    assert.deepEqual(I18n.getData(), { custom: customLanguagePack }, 'I18n data should have custom language pack');
+    assert.deepEqual(I18n.getData().custom, customLanguagePack, 'I18n data should have custom language pack');
```

Comment:
This was flaking since there is a test that adds an Arabic language pack to verify that rtl mode is detected. So I changes this to be more explicit and only check for the custom language pack.

Reply:
Thanks for this! I wonder if we need to add a way to clean up the I18n data between tests 🤔

Reply:
Maybe the resource thing required it since it was using the same key. This uses different keys so less of an issue to have a unload language pack here.